dice.js

```diff
@@ -0,0 +1,26 @@
+const generateNumber = function(n) {
+  n = n ? n : 6;
```

Comment:
Great solution; Also you can allow named parameters to be initialized with default values if no value or undefined is passed:
```const generateNumber = function(n=6){...}```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters